cmd/cosign/cli/sign/sign_blob.go

```diff
@@ -82,6 +98,20 @@ func SignBlobCmd(ro *options.RootOptions, ko options.KeyOpts, regOpts options.Re
 		signedPayload.Bundle = cbundle.EntryToBundle(entry)
 	}
 
+	// if bundle is specified, just do that and ignore the rest
+	if ko.RFC3161TimestampPath != "" {
+		signedPayload.Base64Signature = base64.StdEncoding.EncodeToString(sig)
+
+		contents, err := json.Marshal(signedPayload)
```

Comment:
Shouldn't this be writing `signedPayload.RFC3161Timestamp`, not `signedPayload.Base64Signature`?